src/dune_engine/build_system.ml

```diff
@@ -273,7 +272,7 @@ end = struct
       , Dep.Facts.digest facts ~env
       , target_paths rule.targets.files @ target_paths rule.targets.dirs
       , Action.for_shell action
-      , can_go_in_shared_cache
+      , true (* TODO: don't forget to remove this *)
```

Comment:
TODO comment intentionnaly left here, as removing the `true` in the tuple changes the digest, and trips the tests. I wanted to make a separate commit that only removes this line and changes the hash to make it clean for the CI.